package.json

```diff
@@ -44,7 +44,7 @@
   },
   "homepage": "https://github.com/anthonyjgrove/react-google-login",
   "dependencies": {
-    "@types/react": "*",
+    "@types/react": "^16.9.11",
```

Comment:
why did you change the `*` ?

Reply:
Not sure how that happened, I reverted it back